scripts/release.sh

```diff
@@ -164,7 +192,15 @@ if [[ $? == 1 ]]; then
 
     # This runs Jazzy to generate the documentation.
     echo "Running Jazzy to generate documentation..."
-    jazzy --clean --objc --framework-root SmartDeviceLink --sdk iphonesimulator --umbrella-header SmartDeviceLink/public/SmartDeviceLink.h --theme theme --output docs
+    # generate-documentation.sh throws an error if we are not in the correct directorywhen we run it.
+    cd scripts
+    chmod u+x ./generate-documentation.sh
+    if [ $(uname -m) == "x86_64" ]; then
+        ./generate-documentation.sh
+    else
+        arch -x86_64 /bin/bash ./scripts/generate-documentation.sh
```

Comment:
Why does one call `./generate-documentation.sh` and the other `./scripts/generate-documentation.sh`?

EDIT: Yup, looks like it fails on M1
<img width="728" alt="Screen Shot 2022-07-28 at 9 24 01 AM" src="https://user-images.githubusercontent.com/1767063/181516220-ab6b0dfe-297f-49ff-941c-c0f2ed6d869f.png">

Reply:
it looks like I added the cd to shorten the paths and then failed to fix both paths.  good catch.